site2/docs/functions-develop-api.md

```diff
@@ -102,6 +102,13 @@ public class ExclamationFunction implements Function<String, String> {
 
 For more details, see [code example](https://github.com/apache/pulsar/blob/master/pulsar-functions/java-examples/src/main/java/org/apache/pulsar/functions/api/examples/ExclamationFunction.java).
 
+:::note
+
+The return type of the `Function` can be wrapped in a Pulsar `Record` generic which gives more control on the output message (topic, schema, properties, ...)
+You can use the `Context::newOutputRecordBuilder` method to build this `Record` output.
```

Comment:
Does it make sense to provide at least an example to show users how to use this method and what they can get?

Reply:
Yes, sure. Is `functions-develop-tutorial` the page where it could be added ?

Reply:
Yes, if you have a specific use case to write a function for because the current tutorial page is use-case-based.
Or you can create a new topic under the "Develop" section to introduce the new method/context.

Reply:
I can do the page in the Develop section. But currently it's only available in Java.

Reply:
It's really related to the Java API so I've put the example in the Java tab of this page.